README.rst

```diff
@@ -45,6 +45,17 @@ To install [which is optional], use ::
 
     $ make install
 
+To run Veles you we recommend creating virtualenv for veles::
```

Comment:
Maybe add "built from repo"? Otherwise it sounds like a general instructions for everyone wanting to run Veles.